python/feathub/feature_tables/sources/kafka_source.py

```diff
@@ -115,7 +117,26 @@ def __init__(
             raise FeathubException(
                 "startup_datetime is required when startup_mode is timestamp."
             )
-        self.is_bounded = is_bounded
+
+    def get_bounded_view(self) -> TableDescriptor:
+        return KafkaSource(
```

Comment:
Can we use `deepcopy(self)` similar to what this PR do in `FeatureView::get_bounded_view(...)`?